stdlib/tracemalloc.pyi

```diff
@@ -41,6 +41,15 @@ class Frame:
     filename: str
     lineno: int
     def __init__(self, frame: _FrameTupleT) -> None: ...
+    def __lt__(self, other: Frame) -> bool: ...
```

Comment:
Confirmed that `__lt__` for some reason does not have the NotImplemented trick.

Reply:
This is why I used the `sys.version_info` guards, FWIW: https://github.com/python/cpython/pull/30818